docs/troubleshooting.md

```diff
@@ -60,18 +61,18 @@ If the challenge is a pwnable, you might see an error formatted like this:
 [E][2020-02-02T20:20:02+0200][1] void subproc::subprocNewProc(nsjconf_t*, int, int, int, int)():204 execve('/bin/sh') failed: No such file or directory
 ```
 
-That's an NsJail error. In this case, it is complaining that `/bin/sh` failed and gave the error *"No such file or directory"*.
+That's an nsjail error. In this case, it is complaining that `/bin/sh` failed and returned the error *No such file or directory*.
```

Comment:
I also think nsjail looks better in lowercase (and the command is written this way) but just fyi https://github.com/google/nsjail uses the NsJail capitalization.
But still, +1 for lower case :)

Reply:
Thanks for the info! I agree with sticking with lower case here; as long as the kCTF  doc is consistent, it's fine. When reviewing the Google Cloud walkthrough with Eduardo, we also opted for the lower case spelling.